README.md

```diff
@@ -119,6 +119,8 @@ Additionally, `cowboy_swagger` can be configured/customized from a `*.config` fi
  %% cowboy_swagger config
  {cowboy_swagger,
   [
+   %% `path`: Path where you can access Swagger-UI. Default: `/api-docs`
+   {path, "/api-docs-swagger"},
```

Comment:
It says the default is `/api-docs` but we seem to be changing that. Do we need to change the default? I'm missing something, as I don't understand this particular change.